package.json

```diff
@@ -69,7 +71,7 @@
     "stylelint-scss": "5.0.1",
     "typescript": "5.1.6",
     "util": "0.12.5",
-    "web3": "^4.0.3",
+    "web3": "^1.10.0",
```

Comment:
I this says ^1.10.0 then it will not force 4.0.3 to downgrade. You should use either ~1.10.0 or 1.10.x. It will cause devs to have to `rm -rf node_modules && yarn` in some cases.